src/buildstream/storage/directory.py

```diff
@@ -466,6 +466,13 @@ def _import_files(
     def _export_files(self, to_directory: str, *, can_link: bool = False, can_destroy: bool = False) -> None:
         raise NotImplementedError()
 
+    # _ensure()
+    #
+    # Makes sure the files for the directory are available locally. Should be called before
+    # using _get_underlying_path()
+    def _ensure(self):
```

Comment:
Can we make the method name a bit clearer? Maybe `_ensure_local()`?

Reply:
Sure. Fixed.